### ISSUE TITLE
[Bug]: Screen reader not announcing selected state in pivot component

### ISSUE THREAD

**Issue by parjai25** [microsoft/fluentui#23016]
### Library

Web Components (@fluentui/web-components)

### System Info

```shell
Window 11, Browser- Chrome
```

### Reproduction

https://codepen.io/parjai25/pen/dydNzNL

## Repro steps

https://developer.microsoft.com/en-us/fluentui#/controls/web/pivot

Step 1: Open above URL.
Step 2: Navigate to any tab of pivot item and activate it.
Step 3: Navigate to some another tab and activate it.
Step 4: Navigate to selected tab item using left/right arrow key.
Step 5: Verify that Screen reader does not announce selected state on selected tabs items.

## Actual Behavior
Screen reader does not announce state as 'selected ' on selected tabs items. Screen reader announces tab item name only. E.g screen reader announces '<tab name> tab item'

## Expected Behavior
Screen reader should announce state as 'selected ' on selected tabs items. Screen reader should announce '<tabName> tab item selected'.

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by adelKhatib** [comment]
adding that this bug also repro on **Edge Browser.**

for the provided codepen:

**Actual Result:**
Screen reader is announcing “Recent tab item 2 of 3” when user activates "Recent" tab item.
 
**Expected result:** 
Screen reader should announce **selected state info** along with “Recent tab item 2 of 3” when user activates "Recent" tab.

**Comment by zayedrasha** [comment]
Any updates on this bug?
